content/docs/start/model-management/model-registry.md

```diff
@@ -0,0 +1,294 @@
+---
+title: 'Get Started: Model registry'
+description:
+  'Set up a Git-based model registry with DVC to track and manage models, their
+  versions and lifecycle stages.'
+---
+
+# Get Started: Model Registry
+
+Just as we use experiment tracking to manage model development, it is a good
+idea to keep a <abbr>model registry</abbr> to manage the lifecycle of the models
+we get from our experiments. Using DVC and [DVC Studio](/doc/studio) we will set
+up a model registry where we can discover, share, deploy and audit all our
+models and which will serve as the single source of truth for our model
+management.
+
+<admon type="tip" id="GTO-tip">
+
+Behind the scenes, DVC Studio uses a command line tool called
+[GTO](<(/doc/gto)>) for most model registry actions.
+
+With GTO you can also set up the model registry locally without DVC Studio (some
+functionality is exclusive to DVC Studio, however). You can see how this is done
+in the expandable "Under the hood" sections in this chapter.
```

Comment:
```suggestion
With GTO you can also set up the model registry locally without DVC Studio.
You can see how this is done in the expandable "Under the hood" sections in this chapter.
```

Minor, but I would drop the parenthetical here. I would rather point this out where it applies than make this general statement that leaves users wondering what functionality is exclusive to Studio.

Reply:
Agreed